github/github.go

```diff
@@ -311,7 +316,6 @@ func (c *Client) NewRequest(method, urlStr string, body interface{}) (*http.Requ
 			return nil, err
 		}
 	}
-
```

Comment:
Was this line removed intentionally?

It doesn't look like an improvement, so unless you have a reason, I'd suggest reverting it.

Reply:
yup, stray line kill . Thanks for catching that!